Makefile.dapper

```diff
@@ -12,6 +12,12 @@ export MAKEFLAGS
 export PROJECT
 export USING
 
+# We need the host's actual IP to connect to the nexodus service, otherwise it wont work
+ifneq (,$(filter nexodus,$(USING)))
+HOST_IP = $(shell ip route get 1 | grep -o "src .*" | cut -f2 -d' ')
```

Comment:
`ip` can produce JSON output:

```suggestion
HOST_IP = $(shell ip --json route get 1 | jq -r '.[0].prefsrc)
```

Reply:
Since this is in `Makefile.dapper` which runs outside the container, I wonder if we should rely on `jq` which forces everyone to have it installed.
Going with `grep | cut` should work for everyone as those are quite ubiquitous, unlike `jq`.

Reply:
Fair enough, let’s leave it as-is.